### DISCUSSION TITLE
Codigo compatible con otro navegador

### DISCUSSION THREAD

**Discussion by KarymeToxqui** [Launch-X-Latam/MisionFrontEnd#171] [Q&A] [answered]
Hola, alguien sabe como hacer que el código HTML se pueda abrir en opera o tengo que descargar otro navegador?

**Comment by EduGarcia502** [comment]
El codigo html es universal, en rara ocacion es que falla en los navegadoresn

**Reply by KarymeToxqui** [reply]
no me abre ni con chrome, nada mas me abre bien con firefox

**Comment by iam-leo** [comment, marked as answer]
Hola explorer! Mientras tu archivo tenga extensión .html solo arrastras el archivo al navegador que quieras y lo sueltas!

**Reply by KarymeToxqui** [reply]
Muchas Gracias! si me funciono ya! jaja

**Reply by iam-leo** [reply]
De nada! Ahora a practicar :wink: :wink: :wink:
